scripts/python_scripts/subquery_api.py

```diff
@@ -0,0 +1,127 @@
+from typing import List
+import requests
+import json
+
+
+class DeploymentInstance():
+
+    def __init__(self, **kwargs) -> None:
+
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+
+class SubQueryProject():
+
+    deployments = List[DeploymentInstance]
+
+    def __init__(self, **kwargs) -> None:
+
+        self.deployments = []
+        for key, value in kwargs.items():
+            if key == 'deployments':
+                deployments = []
+                for deployment in value:
+                    deployments.append(DeploymentInstance(**deployment))
+                setattr(self, key, deployments)
+
+            setattr(self, key, value)
```

Comment:
Same here - is there any particular reason to use `setattr` here?

Reply:
fixed, thanks